core/src/main/java/tc/oc/pgm/kits/ClearItemsKit.java

```diff
@@ -45,6 +47,13 @@ public void applyPostEvent(MatchPlayer player, boolean force, List<ItemStack> di
     if (this.items) {
       player.getBukkit().getInventory().clear();
       player.getBukkit().getOpenInventory().setCursor(null);
+      InventoryView openInventory = player.getBukkit().getOpenInventory();
+      if (openInventory != null) {
+        Inventory topInventory = openInventory.getTopInventory();
```

Comment:
Is this going to clear a whole chest if you have one open and a clear is applied?

Reply:
Sorry for the slow update.

I've modified this so it will only clear crafting grids. I've included full 3x3 crafting benches, since you can't store items in them normally so it would be weird if you could sneak items through one. It should no longer clear chests, though.

I don't have a good map to test this on, but it does successfully clear 2x2 crafting grids. If we want to test this more thoroughly let me know.